### PR TITLE
Add safety returns after ajaxRender calls

### PR DESCRIPTION
**Pull request by jocel1**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add safety returns after call to ajaxRender()
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Test pages with ajax

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9122)

### CODE REVIEW

File: controllers/front/AddressController.php [hunk 1]
```diff
@@ -138,5 +138,7 @@ public function displayAjaxAddressForm()
                 $addressForm->getTemplateVariables()
             ),
         )));
+
+        return;
```
Comment (Quetzacoalt91) [inline controllers/front/AddressController.php:142]: I'm not sure this one was useful. :)

**Comment by Quetzacoalt91** [comment]
I have quickly checked the changes, it seems not all return statements are needed.

Can you double check please?

**Comment by jocel1** [comment]
@Quetzacoalt91 Actually only one return is needed, but I added extra return everywhere to prevent someone from adding code after this call without thinking about it :)

**Review by eternoendless — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]
Alrighty then!

File: controllers/front/ChangeCurrencyController.php [hunk 2]
```diff
@@ -36,7 +36,11 @@ public function initContent()
         if (Validate::isLoadedObject($currency) && !$currency->deleted) {
             $this->context->cookie->id_currency = (int)$currency->id;
             $this->ajaxRender('1');
+
+            return;
         }
         $this->ajaxRender('0');
```
Comment (PierreRambaud) [inline controllers/front/ChangeCurrencyController.php:42]: Could be `return $this->ajaxRender('0'); ` if you prevent someone from adding code :) Same for every `return;`, just return the latest function called?

Comment (jocel1) [reply controllers/front/ChangeCurrencyController.php:42]: yes but Phpstorm will complain you return from a function returning void :)

**Pull request merged**